tools/xmlschema.rb

```diff
@@ -176,27 +207,24 @@ def printAttribute(_file, _spaces, _attr)
 # \param[in] _elem The SDF element to convert to an xml schema.
 def printXSD(_file, _spaces, _elem)
 
-  if !_elem.elements["description"].nil? &&
-     !_elem.elements["description"].text.nil?
-    printDocumentation(_file, _spaces, _elem.elements["description"].text)
-  end
-
-  _file.printf("%*s<xsd:include schemaLocation='http://sdformat.org/schemas/types.xsd'/>\n", _spaces, "")
-
-  # Print the inclues for the complex type
-  # The includes must appear first
-  _elem.get_elements("include").each do |inc|
-    printInclude(_file, _spaces, inc);
-  end
-
   if _elem.get_elements("element").size > 0 ||
      _elem.get_elements("attribute").size > 0 ||
      _elem.get_elements("include").size > 0
 
     # Print the complex type with a name
     _file.printf("%*s<xsd:element name='%s'>\n", _spaces, "",
                  _elem.attributes["name"])
-    _file.printf("%*s<xsd:complexType>\n", _spaces+2, "")
+
+    if !_elem.elements["description"].nil? &&
+      !_elem.elements["description"].text.nil?
+     printDocumentation(_file, _spaces+2, _elem.elements["description"].text)
+    end
+
+    if _elem.attributes['name'] == "pose"
```

Comment:
If we use `printElem` as I suggested, I'm not sure if we need a special treatment of `pose` here.

Reply:
removed 75b33b6d6447182b5945832d8296cf883f55c908